pack/fhv/fhvc.json

```diff
@@ -283,6 +283,174 @@
+    "code": "10689",
+    "doom": 5,
+    "double_sided": true,
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "flavor": "You must gather Mother Rachel's flock, willing or not, to their place at the table.",
+    "illustrator": "Rob Laskey",
+    "name": "Lambs to the Slaughter",
+    "pack_code": "fhvc",
+    "position": 189,
+    "quantity": 1,
+    "stage": 3,
+    "text": "You cannot trigger [codex] codex entries.\n[reaction] After you evade a [[Resident]] enemy by 2 or more: Add them to the victory display.\n<b>Objective</b> – if 1 plus 1 [per_investigator] [[Resident]] cards are in the victory display, <b>(→R3)</b> <i>(page 60)</i>.",
+    "type_code": "agenda"
+  },
+  {
+    "code": "10692",
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 7,
+    "enemy_damage": 1,
+    "enemy_evade": 2,
+    "enemy_fight": 2,
+    "faction_code": "mythos",
+    "health": 2,
+    "illustrator": "Lin Hsiang",
+    "name": "Frenzied Reveler",
+    "pack_code": "fhvc",
+    "position": 192,
+    "quantity": 2,
+    "text": "<b>Forced</b> – When Frenzied Reveler is defeated: Place 1 doom on the agenda. This effect can cause the current agenda to advance.\n[action]: <b>Parley</b> – Test [agility] (4). If you succeed, remove Frenzied Reveler from the game.",
+    "traits": "Humanoid. Resident.",
+    "type_code": "enemy"
+  },
+  {
+    "back_link": "01690b",
+    "clues": 0,
+    "code": "10690a",
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 5,
+    "faction_code": "mythos",
+    "flavor": "Dozens of visitors gather around the base of the effigy.",
+    "illustrator": "Mauro Dal Bo",
+    "name": "The Crossroads",
+    "pack_code": "fhvc",
+    "position": "190a",
+    "quantity": 1,
+    "shroud": 3,
+    "subname": "Morning [day]",
+    "text": "[action]: You ask the locals about the Vale <b>([codex] 10)</b>.\n[fast] If there are exactly 1 or 2 investigators in the game, during your turn: <b>Move</b> <i>(to a connection location)</i>. (Group limit once per round.)",
+    "traits": "Hemlock Vale. Central.",
+    "type_code": "location"
+  },
+  {
+    "clues": 0,
+    "code": "10690b",
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 5,
+    "faction_code": "mythos",
+    "flavor": "A crowd of revelers dance around a roaring bonfire. Mother Rachel stands before it, smiling broadly.",
+    "illustrator": "Mauro Dal Bo",
```

Comment:
`"hidden": true,`

Please add the above as its own line. For cards that are split into two, we mark the back as hidden, because otherwise the DB gets very confused about two cards existing in what is the same slot. Not ideal, I know, but that's currently how it works.